cmd/snappy/cmd_update.go

```diff
@@ -53,7 +53,8 @@ const (
 	shutdownTimeout = "+10"
 )
 
-var shutdownMsg = i18n.G("snappy autopilot triggered a reboot to boot into an up to date system -- temprorarily disable the reboot by running 'sudo shutdown -c'")
+// TRANSLATORS: please keep this under 80 characters if possible
+var shutdownMsg = i18n.G("Snappy needs to reboot to finish an update. Defer with 'shutdown -c'.")
```

Comment:
Hm, minor nitpick, "sudo shutdown -c" was used in the old one because some people tripped over it.